sdcm/db_stats.py

```diff
@@ -413,6 +430,42 @@ def calculate_stats_average(self):
         if total_stats:
             self._stats['results']['stats_total'] = total_stats
 
+    def _cut_comments(self, result):
+        return '\n'.join([i.strip() for i in result.split('\n') if not i.startswith('#')])
+
+    def build_archive(self):
+        download_files = ['/proc/meminfo', '/proc/cpuinfo', '/proc/interrupts', '/proc/vmstat']
+        storing_dir = os.path.join(self.db_cluster.logdir, 'tmp')
+        os.mkdir(storing_dir)
+        for node in self.db_cluster.nodes:
+            src_dir = node.prepare_file_for_archive(download_files)
+            node.receive_files(src=src_dir, dst=storing_dir)
+            with open(os.path.join(storing_dir, node.name, 'installed_pkgs'), 'w') as pkg_list_file:
+                pkg_list_file.write(node.get_installed_packets())
+        self.log.info('Creating archive....')
+        archive = shutil.make_archive(self.db_cluster.logdir, 'zip', root_dir=storing_dir)
+        self.log.info('Path to archive file: %s' % archive)
+        return archive
+
+    @staticmethod
+    def generate_s3_url(file_path):
```

Comment:
Alex, we discussed this, copy-paste of the code is BAD practice!
If you see that the code can be generalized you are more than encouraged to do this.
I think that the best place for this code is in `utils.py`

Reply:
Move to utils.py.
Create as Class with static methods. Need your opinion